src/trans/gpu/internal/spnsde_mod.F90

```diff
@@ -103,7 +105,7 @@ SUBROUTINE SPNSDE(KF_SCALARS,PEPSNM,PF,PNSD)
 #ifdef OMPGPU
   !$OMP TARGET TEAMS DISTRIBUTE PARALLEL DO
   !! DEFAULT(NONE) PRIVATE(IJ) &
-  !!$OMP&   SHARED(KM,F,ZN,ZEPSNM,KMLOC)
+  !!$OMP&   SHARED(KM,ZN,ZEPSNM,KMLOC)
```

Comment:
`ZEPSNM` isn't used -> I guess you left this in because you thought it might be needed for OpenMP offload for some reason?

Reply:
I believe this is an artifact from earlier times. Turns out that PEPSNM is passed as an argument, I think there was a time when this was probably not passed as an argument. I removed it and added PEPSNM to the clause, same as OpenACC.